### PR TITLE
Modified update-to-head.sh and adding nightly-ci-run and cron-nightly-ci-run to run on our qe cluster

### PR DESCRIPTION
**Pull request by rupalibehera**
1. modified update-to-head script to keep in sync with the operator one
2. Adding pipelinerun.yaml and cron.yaml, which will execute the pipelinerun

### CODE REVIEW

File: openshift/release/nightly-ci-run.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            - name: create-pr
+              workingDir: $(workspaces.source.path)
+              env:
+                - name: HUB_VERSION
+                  value: "true"
+                - name: GITHUB_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: nightly-ci-github-hub-token
+                      key: hub-token
```
Comment (savitaashture) [inline openshift/release/nightly-ci-run.yaml:45]: From where do we get this secret :thinking:

File: openshift/release/cron-nightly-ci-run.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 1
+  concurrencyPolicy: Replace
+  schedule: "0 0 * * *"
+  startingDeadlineSeconds: 200
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: nightly-ci-sa
```
Comment (savitaashture) [inline openshift/release/cron-nightly-ci-run.yaml:15]: where are we creating this SA :thinking:

File: openshift/release/nightly-ci-run.yaml (hunk 1 above)
Comment (rupalibehera) [reply openshift/release/nightly-ci-run.yaml:45]: this is there on the qe cluster

File: openshift/release/cron-nightly-ci-run.yaml (hunk 2 above)
Comment (rupalibehera) [reply openshift/release/cron-nightly-ci-run.yaml:15]: we have already created the sa on the qe cluster

**Review by savitaashture — COMMENTED** [review]
:+1: :tada: :+1: 

/lgtm

**Pull request merged**